src/content/defi/index.md

```diff
@@ -21,8 +21,8 @@ Defi stands out from traditional financial services in several significant ways:
 
 - **Self custody** - Users maintain custody of their assets, guaranteeing access and keeping control for themselves
 - **Open access** - Decentralized finance operates across border, providing open access to financial services without geographical restrictions
-- **Transparent** - The code underlying decentralized finance is open sourced, allowing anyone to inspect how these financial systems operate
-- **Composable** - Financial primatives, such as lending, can be created and built on to create new applications
+- **Transparent** - The code underlying decentralized finance is open sourced, allowing anyone to inspect how these financial systems operate and where funds are moving
+- **Composable** - Financial primatives, such as lending, can iterated on to create new applications
```

Comment:
```suggestion
- **Composable** - Financial services, such as lending, can be iterated on to create new applications
```
was this grammatically correct?

Reply:
This could work... I think "primitives" makes sense here for the "Composable" category, since we're talking about composing individual units (or "primitives") to create more complex applications... but I can also see how the term may end up confusing some readers.

(If we go with "primitives" let's just correct the spelling)

In either case, "primitives" or "services", I think "iterated on" is not the best... we're talking about composability, as in "money Lego bricks"... I think something emphasizing that these can be built on top of, or mixed together with new/other smart contracts.